airbyte-webapp/src/components/connection/ConnectionForm/Section.module.scss

```diff
@@ -16,3 +19,20 @@
     gap: 0;
   }
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+
+  .arrow {
+    font-size: $icon-size;
+    line-height: $icon-size;
+    color: colors.$dark-blue;
+    transition: 0.3s;
+    cursor: pointer;
+
+    &.collapsed {
+      transform: rotate(90deg);
```

Comment:
The chevron rotation is different from what we see in the connection status page:

Collapsed: `>`
Open: `V`

Reply:
fixed